pennylane/vqe/vqe.py

```diff
@@ -175,6 +253,31 @@ def wires(self):
     def name(self):
         return "Hamiltonian"
 
+    @property
+    def grouping_indices(self):
+        """Return the grouping indices attribtue.
+
+        Returns:
+            list[list[int]]: indices needed to form groups of commuting observables
+        """
+        return self._grouping_indices
+
+    def compute_grouping(self, grouping_type="qwc", method="rlf"):
```

Comment:
There is a degeneracy in the naming here, since `compute_grouping` is also the name of the input parameter of the Hamiltonian. The doc strings below are a bit confusing because they refer also to this variable. Would it be worth renaming one of them? e.g., `compute_groups` for the function`, or something even more different?

Reply:
Oh good catch, these references were a matter of copy-paste errors...they have no meaning here. 

I'd like to leave the name conflation because the two methods do exactly the same thing, they fill the attribute...